### DISCUSSION TITLE
Event type <class 'nonebot.adapters.onebot.v11.event.PrivateMessageEvent'> not match annotation GroupMessageEvent, ignored

### DISCUSSION THREAD

**Discussion by XCCdesu** [nonebot/discussions#131] [问答]
小白 使用haruka_bot 的时候显示Event type <class 'nonebot.adapters.onebot.v11.event.PrivateMessageEvent'> not match annotation GroupMessageEvent, ignored 查文档也不知道怎么解决

**Comment by yanyongyu** [comment]
这个是正常的日志，表示收到了私聊消息，预期响应为群消息。调整日志等级为INFO即可忽略。
